api/filter/http/gzip.proto

```diff
@@ -55,32 +48,31 @@ message Gzip {
   // "DEFAULT" if not specified.
   CompressionStrategy compression_strategy = 4 [(validate.rules).enum.defined_only = true];
 
-  // Array of strings that allows specifying which "cache-control" header
-  // values yield compression. Normally, if "cache-control" is present in the response headers,
-  // compression should only occur if directives indicate that the content should not be cached;
-  // e.g. no-cache or no-store.
-  repeated string cache_control = 5
-      [(validate.rules).repeated = {min_items: 0, max_items: 10, unique: true}];
-
-  // Array of strings that allows specifying which mime-types yield compression; e.g.
+  // Set of strings that allows specifying which mime-types yield compression; e.g.
   // application/json, text/html, etc. When this field is not specified, compression will be applied
-  // to any "content-type".
-  repeated string content_type = 6 [(validate.rules).repeated = {min_items: 0, max_items: 30}];
+  // to the following MIME-TYPES: html, text, css, js, json, svg, xml. xhtml".
+  repeated string content_type = 6 [(validate.rules).repeated = {min_items: 0, max_items: 50}];
 
-  // Allows disabling compression if response contains "etag" (entity tag)
-  // header. Default is false.
+  // Value that disables compression if response contains Etag (entity tag)
+  // header. This option is disabled by default (false), which means that strong Etags will be
+  // re-writen to its weak form before being dispatched to the client. If for some reason the client
+  // does not support weak validation, it is recommended to have this enabled.
```

Comment:
per style guide PR, since this is not considered implemented yet can we change all of the `BoolValue` to `bool` in cases where the default is false? (In general prefer primitive type of the default is already what you want).